_data/changelogs/component-table.yml

```
@@ -4,7 +4,7 @@ type: component
 changelogURL:
 
 items:
-  - date: NNNN-NN-NN
+  - date: 2023-06-17
```

Is this the accurate date for the merge? The PR shows it was merged [two days ago](https://github.com/uswds/uswds-site/pull/2143#event-9838545393).
```suggestion
  - date: 2023-07-17
```